### PR TITLE
Docs: warm about password sharing links with the same password

### PR DESCRIPTION
**Pull request by stsewd**
---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--11525.org.readthedocs.build/en/11525/

- Developer's documentation (`dev`): https://dev--11525.org.readthedocs.build/en/11525/

### CODE REVIEW

File: docs/user/commercial/sharing.rst [hunk 1]
```diff
@@ -67,6 +67,13 @@ view your project.
    This is useful for when you have documentation you want users to bookmark.
    They can enter a URL directly and enter the password when prompted.
 
+.. warning::
+
+   Since multiple passwords can be created for a single project,
+   each password must be unique within that project.
+   If you generate multiple sharing links using the same password for a project,
+   only the first link will work.
```
Comment (ericholscher) [inline docs/user/commercial/sharing.rst:75]: Can we check against this when we do the hashing, or will they hash differently because of the salt?

**Review by ericholscher — APPROVED** [review]

File: docs/user/commercial/sharing.rst (hunk 1 above)
Comment (stsewd) [reply docs/user/commercial/sharing.rst:75]: We can check, but we will need to check each hashed password, which will be slow if users have lots of passwords, hopefully it's more fast with the new hash algo we are going to implement. So, we could have that feature.

Comment (stsewd) [reply docs/user/commercial/sharing.rst:75]: The form is now able to check for dups with https://github.com/readthedocs/readthedocs-corporate/pull/1849, should we still mention this warning?

Comment (ericholscher) [reply docs/user/commercial/sharing.rst:75]: I think we should be fine closing this, if the form can check for it.

**Pull request closed without merging**